README.md

```diff
@@ -114,13 +114,35 @@ app.mount('#app');
 
 And that's all you need to get started! Have fun!
 
-### Nuxt.js
-This module is not yet compatible with [Nuxt 3](https://v3.nuxtjs.org/getting-started/introduction). As Nuxt 3 is still in development, we recommend using [Nuxt 2](https://nuxtjs.org/guide/getting-started) and [@imgix/vue-imgix](https://github.com/imgix/vue-imgix) v2.x.x. instead.
+### Nuxt.js v3
 
-However, Nuxt.js now has a built-in [Image component](https://image.nuxtjs.org/getting-started/installation) that can be used alongside imgix.
+#### With `nuxt-image`
+
+Nuxt.js now has a built-in [Image component](https://image.nuxtjs.org/getting-started/installation) that can be used alongside imgix.
 
 You can read more about our `nuxt-image` provider and how to use it in the [Nuxt docs](https://image.nuxtjs.org/providers/imgix).
 
+#### Using this library directly
+
+> For Nuxt v2 support, please look at the [`main`](https://github.com/imgix/vue-imgix/tree/main) branch.
```

Comment:
Was this meant to refer to a different branch?

Reply:
I don't think so...

Reply:
I thought nuxt2 maps to vue2 and nuxt3 to vue3, so it sounded like nuxt2 support would be found on a different branch or this line should actually say nuxt3?

Reply:
Right - this PR was made before we made the decision to switch the branches. In the other PRs that switch the branches this is updated